cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -85,6 +92,13 @@ xref:../../cnv/cnv_logging_events_monitoring/cnv-events.adoc#cnv-viewing-vm-even
 
 == Known issues
 
+* If you renamed the custom resource when you deployed an earlier version of
+{CNVProductName}, you cannot upgrade directly to {CNVProductName} {CNVVersion}.
+The custom resource must be named `kubevirt-hyperconverged`, which is the default
+name. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822266[*BZ#1822266*])
+** As a workaround, delete the existing custom resource and create a new one.
```

Comment:
Sorry, I'd suggest to rename the existing one to the new name (this is allowed!) or create a new one with the right name and only after that delete the original CR.